docs/services/slam/cartographer/_index.md

```diff
@@ -357,125 +453,66 @@ See [Attributes](#attributes) for details.
+   {{% /alert %}}
 
-Once the session has ended, the map will be saved to your **Location** page's **SLAM library** tab.
+## Create a new map from cloud data
 
-![offline mapping available maps](/services/slam/offline-mapping-available-maps.png)
-
-You can click `View map` to view the map in a dynamic pointcloud viewer.
-
-![slam library view map](/services/slam/slam-library-view-map.png)
-
-#### "Update existing map" mode
-
-This is similar to [viewing the map in "create new map" mode](#create-new-map-mode), except it builds on top of an existing map.
-
-{{% alert title="Info" color="info" %}}
-
-Cartographer may take several minutes to find your robot's position on the existing map. In the meantime, your robot will show up at the map's origin (`(x,y)` coordinates `(0,0)`).
-
-You may not want to move your robot until it has localized correctly.
-
-{{% /alert %}}
-
-#### "Localize only" mode
-
-Navigate to the **Control** tab on your robot's page and click on the drop-down menu matching the `name` of the service you created.
-
-Unlike when creating or updating a map, you do not need to start and end a slam session. The pointcloud for the existing map will appear **immediately** and Cartographer will try to find your robot's position on it.
-
-Since the map will not change, nothing new will be added to this robot's location's **SLAM library**.
-
-![slam RC card localize only](/services/slam/slam-RC-card-localize-only.png)
-
-{{% alert title="Info" color="info" %}}
-
-Cartographer may take several minutes to find your robot's position on the existing map. In the meantime, your robot will show up at the map's origin (`(x,y)` coordinates `(0,0)`).
-
-If you move your robot, it will appear to be moving in a trajectory from the map's origin.
-
-{{% /alert %}}
-
-## Offline mode
-
-In this mode, you specify a range of **previously captured** LiDAR and optional IMU data to create a map or update an existing map in the cloud.
+You can specify a range of **previously captured** LiDAR and optional IMU data to create a map or update an existing map in the cloud.
+You can browse your previously captured data from the **Data** page under the **Point clouds** tab (for LiDAR data) and **Sensors** tab (for IMU data).
 
 ### Requirements
 
-You can browse your previously captured data from the **Data** page.
 To create a map, you must have already captured LiDAR data in the location in which you would like to create the map.
 
-Example of previously captured LiDAR data (note, you can click on a row to see the `Robot ID` of the robot the component belonged to):
+The following example shows the previously-captured LiDAR data under the **Point clouds** tab for a robot named `test`.
+Selecting a row opens a pane to the right that contains more information, such as the `Robot ID` of the robot the component belongs to:
```

Comment:
Nice addition!

Also, one of the UI improvements I asked Data/ML for was to show the robot name and ID for the "Sensors" tab, which is why the IMU screenshot doesn't show those: https://viam.atlassian.net/browse/DATA-2066